src/RichText.xml

```diff
@@ -0,0 +1,95 @@
+            <enumerationValues>
+                <enumerationValue key="control">Control</enumerationValue>
+                <enumerationValue key="text">Text</enumerationValue>
+            </enumerationValues>
+        </property>
+        <property key="stringAttribute" type="attribute" required="false">
+            <caption>Value attribute</caption>
+            <category>Data source</category>
+            <description>The attribute that contains the rich text</description>
+            <attributeTypes>
+                <attributeType name="String"/>
+            </attributeTypes>
+        </property>
+        <property key="customOptions" type="object" isList="true" required="false">
+            <caption>Options</caption>
+            <category>Custom</category>
+            <description>Group the editor options you wish to appear</description>
+            <properties>
+                <property key="option" type="enumeration" defaultValue="bold">
+                    <caption>Options</caption>
+                    <category>Options</category>
+                    <description/>
+                    <enumerationValues>
+                        <enumerationValue key="spacer">Spacer</enumerationValue>
+                        <enumerationValue key="bold">Bold</enumerationValue>
+                        <enumerationValue key="italic">Italic</enumerationValue>
+                        <enumerationValue key="underline">Underline</enumerationValue>
+                        <enumerationValue key="headings">Headings</enumerationValue>
+                        <enumerationValue key="orderedList">Ordered list</enumerationValue>
+                        <enumerationValue key="bulletList">Bullet list</enumerationValue>
+                        <enumerationValue key="blockQuote">Block quote</enumerationValue>
+                        <enumerationValue key="codeBlock">Code block</enumerationValue>
+                        <enumerationValue key="subScript">Subscript</enumerationValue>
+                        <enumerationValue key="superScript">Superscript</enumerationValue>
+                        <enumerationValue key="indent">Indent</enumerationValue>
+                        <enumerationValue key="outdent">Outdent</enumerationValue>
+                        <enumerationValue key="direction">Direction</enumerationValue>
+                        <enumerationValue key="textColor">Text color</enumerationValue>
+                        <enumerationValue key="fillColor">Fill color</enumerationValue>
+                        <enumerationValue key="align">Align</enumerationValue>
+                    </enumerationValues>
+                </property>
+            </properties>
+        </property>
+        <property key="onChangeMicroflow" type="microflow" required="false">
+            <caption>On change microflow</caption>
+            <category>Events</category>
+            <description>Microflow to trigger when the editor is changed</description>
```

Comment:
The `the editor` is not changes, the text/content.
Only trigger when value!=value